andela/ah-frontend-sparta
# 164798164 Reset password link

**What does this PR do?**

-  it allows users to reset their passwords

**Description of the tasks to be completed?**

- Add a link to the login form's `forgot your password` to enable them to enter their email.
- Add a UI to enable users to set their new passwords

**How should this be manually tested**?

- Go to the login page on https://authors-haven-dev.herokuapp.com/login
- Click on the `Forgot password?`. 
- This brings up a modal to enter your email.
- Wait for success or error  message.
- Enter your email address and click 'send reset link' button. 
- Check your Email for the message that was sent that contains a reset link.
- Enter your new password, re-enter to confirm the password and click the reset button to reset your password.

**What are the relevant Pivotal Tracker Stories?**

[#164798164](https://www.pivotaltracker.com/n/projects/2320796/stories/164798164)

**Screenshots**

- Sending an Email link

<img width="1436" alt="Screenshot 2019-05-15 at 21 50 00" src="https://user-images.githubusercontent.com/40162086/57801062-91998f00-775b-11e9-8485-dc6d5eec3f50.png">

- Resetting the Password
<img width="1423" alt="Screenshot 2019-05-15 at 21 49 24" src="https://user-images.githubusercontent.com/40162086/57801083-a0804180-775b-11e9-9527-149cd6bc633a.png">
